cfgov/v1/__init__.py

```diff
@@ -15,9 +16,25 @@ def environment(**options):
     env.globals.update({
         'static': staticfiles_storage.url,
         'reverse': reverse,
+        'render_stream_child': render_stream_child
     })
     env.filters.update({
         'slugify': slugify,
         'richtext': wagtailcore_tags.richtext
     })
     return env
+
+
+@contextfunction
+def render_stream_child(context, stream_child):
```

Comment:
its necessary to specifically to streamblocks with richtext field. without it, it prints those stream fields as raw html instead of rendering them.